src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,63 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY_KEY = "supply";
+    private static final String BUY_KEY = "buy";
+    private static final int KEY_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String inputContent = readData(fromFileName);
+        String report = makeReport(inputContent);
+        writeReport(report, toFileName);
+    }
+
+    private String readData(String filePath) {
+        File inputFile = new File(filePath);
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(inputFile));
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/7caba112-dccb-43bd-b48d-e75d0bf1113c)

Reply:
++ , thank you